jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/query/persistence/QueryDefinitionEntity.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.kie.services.impl.query.persistence;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Lob;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+
+import org.jbpm.kie.services.impl.query.SqlQueryDefinition;
+import org.jbpm.services.api.query.model.QueryDefinition;
+import org.jbpm.services.api.query.model.QueryDefinition.Target;
+
+@NamedQueries(value={
+        @NamedQuery(name="getQueryDefinitions", query="select qd from QueryDefinitionEntity qd"),
+        @NamedQuery(name="getQueryDefinitionByName", query="select qd from QueryDefinitionEntity qd where name = :name")
+})
+@Entity
+@Table(name="QueryDefinitionStore", uniqueConstraints={@UniqueConstraint(columnNames="Q_NAME")})
+@SequenceGenerator(name="queryDefIdSeq", sequenceName="QUERY_DEF_ID_SEQ", allocationSize=1)
+public class QueryDefinitionEntity {
+    
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO, generator="queryDefIdSeq")
+    @Column(name = "id")
+    private Long id;
+ 
+    @Lob
+    @Column(name="Q_EXPRESSION", length=65535)
```

Comment:
main idea was that you don't need any other orm file to be added to your app to gain possibility to use entity and its queries in given app. orm files are not automatically discovered while entities are. But since we already have services orm file anyway that needs to be used it can be moved. 

what do you say if I merge it with this and then do post merge clean up to move both QueryDefinition and DeploymentStoreEntry as both exist in jbpm-kie-services. Will leave ContextMappingInfo as runtime manager does not have dedicated orm file. Thoughts?